nativelib/src/main/scala/scala/scalanative/runtime/package.scala

```diff
@@ -119,6 +119,11 @@ package object runtime {
    */
   @noinline def loop(): Unit = ExecutionContext.loop()
 
+  /** Run the runtime's event loop once and return true when internal queue has
+   *  more Runnable to run.
+   */
+  @noinline def loopRunOnce(): Boolean = ExecutionContext.loopRunOnce()
```

Comment:
I guess we might need also to revisit #3144

Reply:
Yes, see my comment on that PR. Any changes that we make to these APIs should be considering that if Scala stdlib introduces a `Scheduler` API then single-threaded Scala Native will be able to implement an event loop that also supports timers / asynchronous sleeping.

- https://github.com/scala-native/scala-native/pull/3144#discussion_r1097757732

Reply:
@WojciechMazur idea of this PR is kind of shortcut for #3144 which is much better, but discussion for this API was here for years.

So, I've tried minimal hack to avoid very ugly code to play with raw-ptr.

If you plan to move forward your PR I'd like to close this one, but I not sure that reimplement it here is the right move.

Rename some method, do not expose it more than needed -- good point, but I suggest to keep it as simple and small as we can.